archinstall/lib/luks.py

```diff
@@ -61,6 +61,8 @@ def encrypt(self, partition, password=None, key_size=512, hash_type='sha512', it
 		with open(key_file, 'wb') as fh:
 			fh.write(password)
 
+		SysCommand(f'bash -c "partprobe"') # Might be redundant
```

Comment:
A method should probably be used to be consistent with the other source code.
```
self.partprobe()
```

Reply:
I was considering it, but since I wasn't sure I'd be calling it in many other places I skipped it. But it would look cleaner, so I'll add it in the next commit.